### PR TITLE
Update organization.md

### PR DESCRIPTION
**Pull request by crpruett**
Final (?) update to the role document.

### CODE REVIEW

File: doc/organization.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+Back-End Development refers to the inner workings--the handling of various kinds of fetch requests and algorithmic solutions. 
 
   -- Chris Pruett
 
-- Database
+## Database
+The Database role is responsible for storing, altering, and removing employee records from the database.
+
   -- Dylan
   -- Joshua Carrier (*)
```
Comment (jcarrier0026) [inline doc/organization.md:18]: What is the (*) for?

File: doc/organization.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
 # Organization
 
-- Front-End Development:
+## Front-End Development:
+Front-End Development refers to the gui components of the project. The look and feel of the project is handled in this role. 
 
   -- Jonasty
```
Comment (jcarrier0026) [inline doc/organization.md]: Should probably change this to his real name (Jonathan Hasty) to match with the other roles.

File: doc/organization.md [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
+## Back-End Development: 
+Back-End Development refers to the inner workings--the handling of various kinds of fetch requests and algorithmic solutions. 
 
   -- Chris Pruett
 
-- Database
+## Database
+The Database role is responsible for storing, altering, and removing employee records from the database.
+
   -- Dylan
```
Comment (jcarrier0026) [inline doc/organization.md]: Again, we should probably use our real/full names (add last name: Eversman).

**Review by jcarrier0026 — CHANGES_REQUESTED** [review]
Fix these few things and I think its good to go. Not trying to be nitpicky, I'm really just getting us some practice reviewing documents and fixing code before the hard coding stuff starts. Just some easy practice.

File: doc/organization.md (hunk 1 above)
Comment (crpruett) [reply doc/organization.md:18]: It's a wildcard marker. Not sure if we agreed on you having a sort of wildcard role.

**Review by jcarrier0026 — APPROVED** [review]
Looks good.

**Pull request merged**